superset-frontend/spec/javascripts/sqllab/fixtures.ts

```diff
@@ -480,6 +480,8 @@ export const initialState = {
       DEFAULT_SQLLAB_LIMIT: 1000,
       SQL_MAX_ROW: 100000,
       DISPLAY_MAX_ROW: 100,
+      SQLALCHEMY_DOCS_URL: 'test_SQLALCHEMY_DOCS_URL',
+      SQLALCHEMY_DISPLAY_TEXT: 'test_SQLALCHEMY_DISPLAY_TEXT',
```

Comment:
wasn't this part of a previous PR?

Reply:
not sure but we wanted to update the fixture to make sure it has all the config items